src/app.js

```diff
@@ -13,7 +13,7 @@ var F = {
 F.util.query = require('./util/query-util');
 F.util.makeSequence = require('./util/make-sequence');
 F.util.run = require('./util/run-util');
-F.util.classForm = require('./util/inherit');
+F.util.classFrom = require('./util/inherit');
 
```

Comment:
The first 3 were basically so the tests could access them. We could browserify the tests too to get away from this, but needed until then.